effectiveOfficeBackend/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:7-jdk11 AS build
+COPY . /home/gradle/src
+WORKDIR /home/gradle/src
+RUN ./gradlew buildFatJar --no-daemon
+
+FROM openjdk:11
+EXPOSE 8080:8080
+WORKDIR /app
+COPY --from=build /home/gradle/src/build/libs/*.jar /app/app.jar
```

Comment:
Because on line 8 you change the context inside the temporary container, using ```WORKDIR /app``` you may not use fully qualified path in further lines. I mean you may change

```dockerfile
COPY --from=build /home/gradle/src/build/libs/*.jar /app/app.jar
```
to
```dockerfile
COPY --from=build /home/gradle/src/build/libs/*.jar app.jar
```

Do the same with ENTRYPOINT